code-review.md

```diff
@@ -0,0 +1,64 @@
+# Code Review Guidelines
+
+Code reviews are important, both to ensure code quality, and to learn from each
+other. To get the most out of it, code authors should voice their expectations
+of a review, and reviewers should state the performed actions and areas of
+focus.
+
+The necessary prerequisite for a *line-by-line* review is that code authors
+make sure that the changes they propose actually make sense and work at a basic
+level. *Corollary: It should never happen that neither the author nor the
+reviewer actually tests the fix.* It is still okay to request feedback on work
```

Comment:
I would rephrase this and explicitly (albeit redundantly) request that unit / integration tests must be run and must pass. --> remove "work at a basic level", add "the unit and integration tests pass".

Also, nobody knows what a corollary is 😉.